src/auto-evo/AutoEvoExploringTool.Export.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Godot;
+using Path = System.IO.Path;
+
+/// <summary>
+///   Partial class: Export
+/// </summary>
+public partial class AutoEvoExploringTool
+{
+    private void ExportWorlds()
+    {
+        var previousWorld = worldsList.FindIndex(w => ReferenceEquals(w, world));
```

Comment:
Why is reference equality used here? Why not just find by the value directly, surely the GameWorld Equals operation is suitable?